### ISSUE TITLE
Reduce sigdigits in parameter range display

### ISSUE THREAD

**Issue by ablaom** [JuliaAI/MLJBase.jl#948]
```julia
julia> range(Float64, :junk, lower=sqrt(2), upper=sqrt(3))
NumericRange(1.4142135623730951 ≤ junk ≤ 1.7320508075688772; origin=1.5731321849709863, unit=0.15891862259789102)
```
I think 4 sigdigits is sufficient.

The display is defined here: https://github.com/JuliaAI/MLJBase.jl/blob/a8e599f302a7233b25973274cc2b5638d604c98c/src/hyperparam/one_dimensional_ranges.jl#L48

**Comment by adarshpalaskar1** [comment]
Hello, can I work on this issue?

I think this change should work:

https://github.com/JuliaAI/MLJBase.jl/blob/a8e599f302a7233b25973274cc2b5638d604c98c/src/hyperparam/one_dimensional_ranges.jl#L33-L34
```
#replacing line 34 with
value = round(getproperty(r, field), sigdigits=4)
```

New output:
```
julia> range(Float64, :junk, lower=sqrt(2), upper=sqrt(3))
NumericRange(1.414 ≤ junk ≤ 1.732; origin=1.573, unit=0.1589)
```

**Comment by ablaom** [comment]
Looks good to me, thanks. PR welcome.

**Comment by ablaom** [comment]
issue assigned to @adarshpalaskar1

**Comment by ablaom** [comment]
Closed by #958
